### ISSUE TITLE
Crash in latest version

### ISSUE THREAD

**Issue by neico** [tvandijck/PremakeExtension#4]
Once you press the `Run Premake` button it asks you to reload the solution, this is when the crash happens.
Also, the button seems to be in a disabled state most of the time, while the solution file has the required entries. (I have yet to find a reliable method to make the button pressable)

Note: the crash is exclusive to running trough the extension, running premake outside VS and then pressing reload works fine.

Crashlog (it's in german, but the general info is there):
> Anwendung: devenv.exe
Frameworkversion: v4.0.30319
Beschreibung: Der Prozess wurde aufgrund einer unbehandelten Ausnahme beendet.
Ausnahmeinformationen: System.Runtime.InteropServices.COMException
   bei Microsoft.VisualStudio.Shell.ThreadHelper.ThrowIfNotOnUIThread(System.String)
   bei PremakeExtension.RunPremake+<>c__DisplayClass16_0.<Execute>b__0(System.Object, System.Diagnostics.DataReceivedEventArgs)
   bei System.Diagnostics.Process.OutputReadNotifyUser(System.String)
   bei System.Diagnostics.AsyncStreamReader.FlushMessageQueue()
   bei System.Diagnostics.AsyncStreamReader.GetLinesFromStringBuilder()
   bei System.Diagnostics.AsyncStreamReader.ReadBuffer(System.IAsyncResult)
   bei System.IO.Stream+ReadWriteTask.InvokeAsyncCallback(System.Object)
   bei System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   bei System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   bei System.IO.Stream+ReadWriteTask.System.Threading.Tasks.ITaskCompletionAction.Invoke(System.Threading.Tasks.Task)
   bei System.Threading.Tasks.Task.FinishContinuations()
   bei System.Threading.Tasks.Task.FinishStageThree()
   bei System.Threading.Tasks.Task.FinishStageTwo()
   bei System.Threading.Tasks.Task.Finish(Boolean)
   bei System.Threading.Tasks.Task.ExecuteWithThreadLocal(System.Threading.Tasks.Task ByRef)
   bei System.Threading.Tasks.Task.ExecuteEntry(Boolean)
   bei System.Threading.Tasks.Task.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   bei System.Threading.ThreadPoolWorkQueue.Dispatch()
   bei System.Threading._ThreadPoolWaitCallback.PerformWaitCallback()

**Comment by tvandijck** [comment]
Can you build it from master? I think I fixed this in the commit just after I made a release when I forgot to push something before I pressed the release button... Or is this still happening on master?

**Comment by neico** [comment]
Yeah according to the latest commit it should work as it fixes exactly what the error complained about...
I can build it myself tough to make sure, so there's that.

Build Warnings:
> ...\PremakeExtension\RunPremake.cs(145,34,145,56): warning VSTHRD010: Accessing "IVsOutputWindowPane" should only be done on the main thread. Call Microsoft.VisualStudio.Shell.ThreadHelper.ThrowIfNotOnUIThread() first.
 ...\PremakeExtension\RunPremake.cs(149,34,149,56): warning VSTHRD010: Accessing "IVsOutputWindowPane" should only be done on the main thread. Call Microsoft.VisualStudio.Shell.ThreadHelper.ThrowIfNotOnUIThread() first.

Build Error:
> (_CopyOutOfDateSourceItemsToOutputDirectoryAlways target) -> 
 C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin\Microsoft.Common.CurrentVersion.targets(4564,5): error MSB3030: Could not copy the file "...\PremakeExtension\LICENSE.txt" because it was not found.

After fixing that error and re-installing the extension it works fine without any issue. Tough I guess those warnings and the error should be addressed and then a new build pushed so that others don't run into the same.
